workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -31,7 +34,9 @@ public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improba
                 .AddComponent(launcher, WorkerUtils.UnityGameLogic)
                 .AddComponent(score, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeSpawner, WorkerUtils.UnityGameLogic)
-                .AddTransformSynchronizationComponents(clientAttribute)
+                .AddComponent(health, WorkerUtils.UnityGameLogic)
+                .AddComponent(shootBullet, WorkerUtils.UnityGameLogic)
+                .AddTransformSynchronizationComponents(WorkerUtils.UnityGameLogic)
```

Comment:
Why exactly do we need this change. This was put here to make playground feel slightly nicer until we get rid of most of it. It will work fine with server side authority but without any csp it will feel irritating to use.

Reply:
just to make the respawning a bit simpler. otherwise we would need another script, command and component that the client is  authoritative over to make it works

Reply:
This just seems like another reason this shouldn't be in playground. When this eventually becomes a proper gym you don't want to have to change behaviour to make an unrelated starter project easier.